.github/workflows/bitcoin.yml

```
@@ -0,0 +1,76 @@
+name: bitcoin-ci
```

```suggestion
name: bitcoin-core-ci
```

Also, could the file be renamed to just `ci.yml` to better represent what it does?